x-pack/plugins/security_solution/public/common/components/event_details/json_view.tsx

```diff
@@ -54,12 +54,14 @@ export const JsonView = React.memo<Props>(({ data }) => {
 JsonView.displayName = 'JsonView';
 
 export const buildJsonView = (data: TimelineEventsDetailsItem[]) =>
-  data.reduce(
-    (accumulator, item) =>
-      set(
-        item.field,
-        Array.isArray(item.originalValue) ? item.originalValue.join() : item.originalValue,
-        accumulator
-      ),
-    {}
-  );
+  data
+    .sort((a, b) => (a.field > b.field ? 1 : b.field > a.field ? -1 : 0))
```

Comment:
👍🏾 Thanks for alphabetizing! `localeCompare` could work here too if you wanted to simplify it `(a, b) => a.field.localeCompare(b.field);`

Reply:
nice, didn't know that one. ill add it if i have to push again, think my build might just go green ✅